app/service.py

```diff
@@ -24,8 +25,53 @@
 from .db import db
 from .authentication import validate_api_key, validate_master_key
 
+titleDoc = "Linka API"
+urlIcon = "https://pbs.twimg.com/profile_images/1344769935004889088/v2e-nR4V_400x400.jpg"
+
+descripcion_spanish = """
+**📃Bienvenida a la Documentación de AireLibre**
+
+Esta documentación proporciona información sobre cómo consumir y utilizar los datos de AireLibre. 
+
+Te invitamos a explorar y contribuir al proyecto para mejorar la conciencia sobre la calidad del aire en nuestra comunidad.
+
+AireLibre es un proyecto de código abierto bajo la licencia AGPLv3
+
+¡Gracias por unirte a nosotros en este esfuerzo por hacer que la información sobre la calidad del aire sea accesible para todos!
+"""
+
+# Inglés
+descripcion_english = """
+**📃 Welcome to AireLibre Documentation**
+
+This documentation provides information on how to consume and utilize AireLibre's data.
+
+We invite you to explore and contribute to the project to enhance awareness of air quality in our community.
+
+AireLibre is an open-source project licensed under AGPLv3.
+
+Thank you for joining us in this effort to make air quality information accessible to everyone!
+"""
+descripcion_combinada = f"{descripcion_spanish.strip()}\n\n{descripcion_english.strip()}"
```

Comment:
I wonder what's the proper way of translating this... I am sure there's a better approach, e.g. something that involved gettext and pot/po files.

Reply:
No tengo claro como corregir este punto, podrías dar una guía o ejemplo ?

<img src="https://media.tenor.com/M0JxNjnIV04AAAAM/peepocomfy-peepo.gif">

Reply:
<img src="https://media.tenor.com/6w1aupPULL4AAAAj/pepe.gif" />

Reply:
Algo cercano a esto... https://sbabashahi.medium.com/add-translation-to-fastapi-with-gettext-a769ae6dd6bb

Muyy por arriba seria asi:

* En vez de hardcodear el mismo string en EN y ES, solo tenes tus strings en EN y despues tenes un dir `./po/` con las traducciones, a.k.a `.po` files.
* Despues necesitas un middleware de fastapi que agarre el locale info de un request y configure las llamada a gettext para que te tire los strings en el idioma "correcto"...

Reply:
Buenas no logre implementar el `getText` pero utilice el mismo enfoque pero utilizando el parametro `Accept-Language:` para cambiar el texto que aparece envase a los idiomas que tengo configurado en su navegador.

Se creo una carpeta dentro de `app` llamada `lang` el mismo contiene las carpetas de los idiomas mapeados cada uno con su archivo `message.py` y utilizando de middleware el `I18nMiddleware` ubicado en **utils/I18nMiddleware.py** para setear las descripciones envase a su idioma priorizado.

En caso de no tener mapeado el idioma por defecto pone en ingles.

### Pruebas:

Navegador configurado con solo un idioma aun no mapeado (Portugués):
![prueba Portuguse](https://github.com/tchx84/linka/assets/57275367/87edd4a3-9ac3-4704-b096-72f4884c1c0f)

Navegador configurado con 2 idiomas mapeado (Español lationamerica,Español):
![prueba es](https://github.com/tchx84/linka/assets/57275367/32444836-8586-4831-aeb2-4f030aa73f5a)

Navegador configurado con 3 idiomas mapeado (Ingles,Español, Español lationamerica):
![pruebaEN,ES](https://github.com/tchx84/linka/assets/57275367/2ceca395-8aa7-4052-96f7-46d6617e5f29)

Navegador configurado con 3 idiomas mapeado (Español, Español lationamerica,Ingles):
![pruebaES,ES,EN](https://github.com/tchx84/linka/assets/57275367/c3d05e46-2967-4832-9d45-6a96fdf7f4ee)


Aguardo feedback.

<img src="https://i.pinimg.com/originals/e7/55/bc/e755bc37531461dbdb6c76d167da6b28.gif" height="70"/>

Reply:
Si no cumple lo esperado podriamos omitir la descripcion  y llevar lo demas ?

Reply:
Sin gettext mejor no meter nada relacionado a i18n, ni hardcodear texto en multiple idiomas. 

Mejor si rompes el PR en cambios independientes, ej solo la parte que agrega que tipo de api key requiere por ahora.